launcher/launcher.go

```diff
@@ -429,6 +433,38 @@ func ensurePort(host string, defaultPort string) string {
 	}
 }
 
+// sets secure grpc port 443 as helper if provided with invalid https:// endpoint and no port
```

Comment:
```suggestion
// sets secure grpc port 443 as helper if provided with valid https:// endpoint and no port
```

Reply:
I reworded this comment to hopefully be more clear